src/lib/Liblog/pbs_log.c

```diff
@@ -715,24 +715,25 @@ log_open_main(char *filename, char *directory, int silent)
 void
 log_err(int errnum, const char *routine, const char *text)
 {
-	char buf[LOG_BUF_SIZE], *errmsg;
+	char buf[LOG_BUF_SIZE] =  {'\0'};
+	char *errmsg;
 	int  i;
 
 	if (errnum == -1) {
 
 #ifdef WIN32
 		LPVOID	lpMsgBuf;
-		int	err = GetLastError();
+		DWORD	err = GetLastError();
 		int		len;
-
+		snprintf(buf, LOG_BUF_SIZE, "Error code: %lu. ", err);
```

Comment:
Now if the message is long, should we save space by reducing "Error code: XXXXXXX." to something like "Err(XXXXX)" ?